tests/Avalonia.Controls.UnitTests/Primitives/TabStripTests.cs

```diff
@@ -32,8 +28,8 @@ public void First_Tab_Should_Be_Selected_By_Default()
 
             target.ApplyTemplate();
 
-            Assert.Equal(0, target.SelectedIndex);
-            Assert.Same(target.Items[0], target.SelectedItem);
+            Assert.Equal(-1, target.SelectedIndex);
+            Assert.Same(null, target.SelectedItem);
```

Comment:
Unfortunately, we can't change default behavior.
TabStrip.SelectionMode default should be changed to match old behavior, even if it's different from ListBox.

Reply:
Likely, you only need to revert `SelectionModeProperty.OverrideDefaultValue<TabStrip>(SelectionMode.AlwaysSelected);` line.